app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -154,9 +159,16 @@ class MainActivity : ThemedActivity() {
                     .getIntArray(DraftsActionsWorker.ERROR_MESSAGE_RESID_KEY)
                     ?.forEach(::showToast)
 
-                val remoteDraftUuid = workInfo.outputData.getString(DraftsActionsWorker.SAVED_DRAFT_UUID_KEY)
+                val remoteDraftUuid = workInfo.outputData.getString(DraftsActionsWorker.DRAFT_UUID_KEY)
                 val associatedMailboxUuid = workInfo.outputData.getString(DraftsActionsWorker.ASSOCIATED_MAILBOX_UUID_KEY)
-                remoteDraftUuid?.let { draftUuid -> showSavedDraftSnackBar(draftUuid, associatedMailboxUuid!!) }
+                val draftAction = workInfo.outputData.getString(DraftsActionsWorker.RESULT_DRAFT_ACTION_KEY)
+                draftAction?.let {
+                    if (draftAction == DraftAction.SAVE.name) {
+                        showSavedDraftSnackBar(remoteDraftUuid!!, associatedMailboxUuid!!)
+                    } else {
+                        showSentDraftSnackBar()
+                    }
+                }
             }
         }
     }
```

Comment:
I've reworked this part.
You can see it here: https://github.com/Infomaniak/android-kMail/pull/986/commits/3aeb1a89e4a541d3281f30d67e99334e0179b84b
But it'll be probably easier to read it in Android Studio.

Reply:
You should update your commit in case it hasn't been pushed yet. And we merge this